module/product/src/Application/Controller/Api/Attribute/UpdateProductAttributeAction.php

```diff
@@ -128,14 +120,6 @@ public function changeDraftAttribute(
 
         $violations = $this->validator->validate(['value' => $value], $constraint);
         if (0 === $violations->count()) {
-            $command = new ChangeProductAttributeCommand(
```

Comment:
It is a bit different to delete the use of a domain command in a given place a bit different to delete it completely. It is worth considering if it is not worth leaving it or if it can be useful.